recipes-ros/mavros/libmavconn_0.17.4.bb

```diff
@@ -0,0 +1,19 @@
+DESCRIPTION = "MAVLink communication library"
+DEPENDS = "mavlink \
```

Comment:
If you want to put each dependency in a separate line, I would propose also to have the first dependency listed as follows:
```
DEPENDS = "\
    first-package \
...
```